Assets/Scripts/PuzzleGenerator.cs

```diff
@@ -9,11 +9,15 @@
 /// </summary>
 public class PuzzleGenerator : MonoBehaviour
 {
+    public const int InputBoxSize = 5;
```

Comment:
Again, not for this PR but something to consider:

You should start planning to move the guts of the input and outbox system into code. 

Identify some key properties that others would use to configure these boxes, and expose them in the editor. Then you can write code around making the boxes work based on those parameters.

Statically placing the slots in the boxes through the editor will be annoying if you need to modify the number of slots.